lib/components/molecules/Accordion/tests/Accordion.test.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { mount } from 'enzyme';
+// eslint-disable-next-line import/named
+import { defaultAccordion } from '../Accordion.mock';
+import Accordion from '../index';
+
+describe('<Accordion />', () => {
+  let AccordionComponent = '';
+
+  beforeEach(() => {
+    AccordionComponent = mount(
```

Comment:
Put the content as well and test for its rendering in right places, trigger the expend collapse